core/src/constants.rs

```diff
@@ -7,6 +9,10 @@ pub const MAX_FEE_BPS: u64 = 10_000;
 pub const MAX_ST_MINTS: usize = 64;
 pub const MAX_VAULTS: usize = 64;
 pub const MAX_OPERATORS: usize = 256;
+pub const MIN_EPOCHS_BEFORE_STALL: u64 = 1;
+pub const MAX_EPOCHS_BEFORE_STALL: u64 = 10;
+pub const MIN_SLOTS_AFTER_CONSENSUS: u64 = 1000;
+pub const MAX_SLOTS_AFTER_CONSENSUS: u64 = 10 * DEFAULT_SLOTS_PER_EPOCH;
```

Comment:
thoughts on this being: `MAX_EPOCHS_BEFORE_STALL * DEFAULT_SLOTS_PER_EPOCH`?

Reply:
These are not really related so I don't think we need to have them associated like this

Reply:
10-4